server.go

```diff
@@ -41,9 +41,11 @@ func main() {
 	log.Printf("HTTP Read Timeout: %s\n", cfg.ReadTimeout)
 	log.Printf("HTTP Write Timeout: %s\n", cfg.WriteTimeout)
 	log.Printf("Function Readiness Probe Enabled: %v\n", cfg.EnableFunctionReadinessProbe)
+	log.Printf("Image Pull Policy: %s\n", cfg.ImagePullPolicy)
```

Comment:
Let's commit this for brevity.

Reply:
Not clear on what is desired here, can you be more specific?